addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -251,15 +271,28 @@ const FontFamilyPickerUserValueWidget = SelectUserValueWidget.extend({
 
         // Remove Google font
         const googleFontIndex = parseInt(ev.target.dataset.fontIndex);
-        const googleFont = this.googleFonts[googleFontIndex];
-        this.googleFonts.splice(googleFontIndex, 1);
+        const isLocalFont = ev.target.dataset.fontLocal;
```

Comment:
Nitpicker: rename data-font-local into data-local-font? :pray:

Reply:
That's how it was first, but then I changed it to match the same format as the index one.
![image](https://user-images.githubusercontent.com/30048408/193051930-d9129f38-8cc1-4a55-8af6-ff782e7962f6.png)

Isn't it better? I don't mind either way, but is your opinion the same since there is a 'font-index' naming ? (not sure if you saw it)
I'll change if yes